chat-plugins/scavengers.js

```diff
@@ -643,6 +652,26 @@ let commands = {
 		}
 		this.privateModCommand(`(${user.name} has set the points awarded for winning an official scavenger hunt to - ${winPoints.map((p, i) => `(${(i + 1)}) ${p}`).join(', ')})`);
 	},
+
+	/**
+	 * Hunt creation tracking
+	 */
+	huntcount: 'huntlogs',
+	huntlogs: function (target, room, user) {
+		if (!this.can('mute', null, room)) return false;
+
+		if (target === 'reset') {
+			if (!this.can('declare', null, room)) return false;
+			HostLeaderboard.reset().write();
+			this.privateModCommand(`(${user.name} has reset the host log leaderbaord.)`);
+
+			return;
+		}
+
+		HostLeaderboard.visualize().then(ladder => {
+			this.sendReply(`|raw|<div class="ladder" style="overflow-y: scroll; max-height: 300px;"><table style="width: 100%"><tr><th>Rank</th><th>Name</th><th>Points</th></tr>${ladder.map(entry => `<tr><td>${entry.rank}</td><td>${Chat.escapeHTML(entry.name)}</td><td>${entry.points}</td></tr>`).join('')}</table></div>`);
```

Comment:
Um... I think you meant ``<table>``, not ``<div>`` here?

... if you meant ``<div>``, then you forgot a ``</div>``.
... if you meant ``<table>``, then you did ``<div>`` instead of ``<table>``